lib/css_parser/parser.rb

```diff
@@ -148,8 +149,16 @@ def add_block!(block, options = {})
         end
       end
 
+      if options[:ignore_import]
       # Remove @import declarations
-      block.gsub!(RE_AT_IMPORT_RULE, '')
+        block.gsub!(RE_AT_IMPORT_RULE, '')
+      end
+
+      # If font-face is allowed and the css block contains font-face
+      # the put all block in one line, css parser cant handle otherwise.
+      if options[:allow_font_face] && !!(block =~ RE_AT_FONTFACE_RULE)
```

Comment:
in !!(block =~ /some_pattern?) you are looking for a boolean return value, you can also rely on the returning of nil(falsy) if there is no match , /r/.match('David') will return nil . This is just an observation, with the !! you achieve a boolean outcome.